### PR TITLE
use defaultBlock for codeAt, balanceAt and stateAt

### PR DESCRIPTION
**Pull request by caktux**
Fixes part of #714

### CODE REVIEW

File: libweb3jsonrpc/WebThreeStubServer.cpp [hunk 1]
```diff
@@ -266,7 +266,7 @@ std::string WebThreeStubServer::shh_addToGroup(std::string const& _group, std::s
 
 std::string WebThreeStubServer::eth_balanceAt(string const& _address)
 {
-	int block = 0;
+	int block = client() ? client()->getDefault() : 0;
 	return toJS(client()->balanceAt(jsToAddress(_address), block));
```
Comment (debris) [inline libweb3jsonrpc/WebThreeStubServer.cpp]: if client() does not exists, client()->balanceAt() does not make any sense.

**Comment by debris** [comment]
@gavofyork can we assume, that client() always exists? if it does not, API is useless anyway and it should throw some exception. it will simplify this "if" statemants.

**Comment by gavofyork** [comment]
A quick look at the code suggests that unguarded use of client() is safe.

**Comment by debris** [comment]
fixed in pr #723

**Pull request closed without merging**